ironfish-cli/src/commands/wallet/transaction/index.ts

```diff
@@ -92,6 +92,10 @@ export class TransactionCommand extends IronfishCommand {
       }
 
       CliUx.ux.table(noteAssetPairs, {
+        noteHash: {
```

Comment:
Probably would be better if note hash wasn't the first in the list. It seems more like a power user item. Maybe even hiding this behind an `--extended` flag like we do in other Oclif CLI tables would be good

Reply:
agreed, we could hide this behind extended. Also you don't need to say "Note Hash" when this is listing notes. If it says, "Hash" it's the hash of the resource being displayed.